command-line-downloader/src/main/java/uk/ac/ebi/ena/app/utils/MenuUtils.java

```diff
@@ -109,28 +114,57 @@ public static boolean isValidEmailAddress(String email) {
         return result;
     }
 
-    public static Map<String, List<String>>  parseAccessions(String accessions) {
+    public static Map<String, List<String>> parseAccessions(String accessions) {
         Map<String, List<String>> accessionDetailsMap = new HashMap<>();
         if (StringUtils.isEmpty(accessions)) {
 
             return accessionDetailsMap;
         } else if (new File(accessions).exists()) {
-            List<String> accessionList =  accsFromFile(accessions);
+            List<String> accessionList = accsFromFile(accessions);
 
-            return CommonUtils.processAccessions(accessionList);
+            return CommonUtils.processAccessions(Objects.requireNonNull(accessionList));
         } else {
             List<String> accessionList = Arrays.asList(accessions.split(","));
 
             return CommonUtils.processAccessions(accessionList);
         }
     }
 
-    @SneakyThrows
     public static List<String> accsFromFile(String inputValues) {
-        Stream<String> lines = Files.lines(Paths.get(inputValues), StandardCharsets.US_ASCII);
+        try {
+            String[] headers = Files.lines(Paths.get(inputValues))
+                    .map(s -> s.split("\t"))
+                    .findFirst()
+                    .orElse(new String[1]);
+
+            String firstColumn = headers[0];
+
+            if (firstColumn == null) {
+                return new ArrayList<>();
+            } else {
+                if (firstColumn.contains(ACCESSION)) {
+                    TsvParserSettings settings = new TsvParserSettings();
+                    settings.getFormat().setLineSeparator("\n");
+                    settings.selectFields(firstColumn);
+
+                    TsvParser parser = new TsvParser(settings);
+                    List<String[]> accessionIdColumn = parser.parseAll(new File(inputValues));
+                    //removing the column header
+                    accessionIdColumn = accessionIdColumn.subList(1, accessionIdColumn.size());
+
+                    return accessionIdColumn.stream().map(aRow -> aRow[0].replace("\"", "").trim())
+                            .collect(Collectors.toList());
+                } else {
+                    List<String> accessionIds = Files.lines(Paths.get(inputValues), StandardCharsets.US_ASCII).collect(Collectors.toList());
```

Comment:
are you handling the case where there's no header row but multiple columns?

Reply:
the else block is for that.. you mean something like below right:

Reply:
DRX264257
DRX264258
DRX264259

Reply:
that's a single column